github/users_projects.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// ListProjects lists the projects for an user.
```

Comment:
nit: Could you please change "an user" to "the specified user"?

Reply:
updated: https://github.com/google/go-github/pull/1295/commits/2f8a9ac24bbf38d656756098254b88002dcd5447